data/README.md

```diff
@@ -2,9 +2,34 @@
 
 A directory for all util and service modules for storing, manipulating and retrieving data
 
-InferGPT uses a Neo4J Knowledge Graph exclusively.
+InferGPT uses a Neo4j Knowledge Graph exclusively.
+
+## Using Neo4j Desktop
+
+If using Neo4j Desktop, this can be downloaded [here](https://neo4j.com/download/)
+
+Once installed follow the steps below:
+
+1. Create new project to host your DBMS.
+2. Create a new database within the project.
+3. Add the username and password to the `.env` file.
```

Comment:
I think it should be specified that you need to create another `.env` file here - might be just me adding the config settings the `backend/.env` but this tripped me up

Reply:
Unless I'm missing something? Just realised the `.example.env` in the backend is the one with the example property values so is the config intended to live there?

Reply:
Moved the `graph db utils` to the `backend/utils/` as this seems to make more sense